cmd/parse_test.go

```diff
@@ -0,0 +1,34 @@
+package cmd_test
```

Comment:
package cmd として private メソッドを公開しないほうが良いと思います。

Reply:
```
1:50 ~/go/src/github.com/ShintaNakama/csv_parse $ make lint                                                                                                                                                        +[feature/create_testcode]
golangci-lint run
cmd/parse_test.go:1:9: package should be `cmd_test` instead of `cmd` (testpackage)
package cmd
        ^
make: *** [lint] Error 1
```